### ISSUE TITLE
📝Create placeholder ShowCard component

### ISSUE THREAD

**Issue by Thenlie** [Thenlie/Streamability#74]
Create a "loading" components to show while data is being fetched for movie cards. This should be the same size and shape of a normal movie card, just greyed out and maybe pulsing. 

## Resources 🔗 

https://www.npmjs.com/package/react-loading-skeleton

**Comment by Thenlie** [comment]
Since this will be the first placeholder created, we will also want to set up the skeleton theme.
```js
import Skeleton, { SkeletonTheme } from 'react-loading-skeleton'

return (
    <SkeletonTheme baseColor="#202020" highlightColor="#444">
        <p>
            <Skeleton count={3} />
        </p>
    </SkeletonTheme>
)
```

**Comment by Thenlie** [comment]
Closed by #287
